db_sql.go

```diff
@@ -807,6 +859,16 @@ func createMigrations() []migration.Migrator {
 			ADD COLUMN createdby BIGINT,
 			ADD COLUMN modified TIMESTAMP,
 			ADD COLUMN modifiedby BIGINT;`,
+
+		// 10. Archive passwords
+		`
+		ALTER TABLE authuserstore  ALTER COLUMN modified SET DEFAULT NOW();	
+		ALTER TABLE authuserpwd  ADD COLUMN created TIMESTAMP DEFAULT NOW();
+		ALTER TABLE authuserpwd  ADD COLUMN updated TIMESTAMP DEFAULT NOW();	
+
+		CREATE TABLE authpwdarchive (id BIGSERIAL PRIMARY KEY, userid BIGINT NOT NULL, password VARCHAR NULL, created TIMESTAMP DEFAULT NOW());
```

Comment:
We don't usually include the explicit "NULL" that is present here in "password VARCHAR NULL".